bokehjs/gulp/tasks/default.coffee

```diff
@@ -1,10 +1,8 @@
-gulp = require "gulp"
+# default - message + brief help
 
-defaultTasks = [
-  "scripts:watch",
-  "scripts:minify:watch",
-  "styles:minify:watch",
-  "develop:watch",
-]
+gulp = require "gulp"
+runSequence = require "run-sequence"
 
-gulp.task "default", defaultTasks, ->
+gulp.task "default", ->
+  console.log('Building BokehJS for developer mode ...')
+  runSequence("scripts:build", "styles:build", "install", "watch")
```

Comment:
I think this needs to be the `scripts` and `styles` tasks so they get minified.  Actually, if you just call `build` I think you're set.

```coffeescript
runSequence "build", "install", "watch"
```